src/Liquidator.sol

```diff
@@ -84,14 +89,36 @@ contract Liquidator is Ownable {
     }
 
     /**
-     * @notice Sets the maximum auction time on the liquidator.
-     * @dev The maximum auction time is the time from starting an auction to
-     * the moment the price of the auction has decreased to 0.
-     * The maxAuctionTime controls the speed of decrease of the auction price.
-     * @param maxAuctionTime_ The new maximum auction time.
+     * @notice Sets the discount rate for the liquidator.
+     * @dev The discount rate is a multiplier that is used to decrease the price of the auction over time.
+     * @param halfLife The new half life
      */
-    function setMaxAuctionTime(uint256 maxAuctionTime_) external onlyOwner {
-        maxAuctionTime = maxAuctionTime_;
+    function setDiscountRate(uint256 halfLife) external onlyOwner {
```

Comment:
Why don't we give the discount rate directly as input?

Reply:
It was easier to set limits